Kickstarter-iOS/AppDelegate.swift

```diff
@@ -135,10 +135,11 @@ internal final class AppDelegate: UIResponder, UIApplicationDelegate {
         self.viewModel.outputs.authorizeForRemoteNotifications
           .observeForUI()
           .observeValues { [weak self] in
-            UNUserNotificationCenter.current().requestAuthorization(options: [.alert, .sound, .badge]) { _ in
-              self?.viewModel.inputs.notificationAuthorizationCompleted()
+            UNUserNotificationCenter.current().requestAuthorization(options: [.alert]) {
+              (isGranted, _) in
+              self?.viewModel.inputs.notificationAuthorizationCompleted(isGranted: isGranted)
```

Comment:
We want to pass in whether authorization was granted or not so that we don't register a user remotely if they don't want to be. 

Note, we don't use `.sound` or `.badge` for iOS 9, so I assume these settings should be the same for iOS 10.